src/modules/uORB/SubscriptionCallback.hpp

```diff
@@ -78,7 +78,7 @@ class SubscriptionCallback : public SubscriptionInterval, public ListNode<Subscr
 			int fd = orb_subscribe_multi(_subscription.get_topic(), _subscription.get_instance());
 
 			// try to register callback again
-			if (_subscription.forceInit()) {
+			if (_subscription.subscribe()) {
```

Comment:
Is there no distinction anymore between trying and forcing to subscribe?

Reply:
Calling subscribe will always attempt to subscribe, or do nothing if already subscribed. If you call update or copy on a Subscription that isn't actually subscribed it will automatically retry subscribing (throttled).